apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.command.ts

```diff
@@ -15,4 +16,9 @@ export class MarkManyNotificationsAsCommand extends EnvironmentWithSubscriber {
   @IsOptional()
   @IsBoolean()
   readonly archived?: boolean;
+
+  @IsOptional()
+  @Type(() => Date)
+  @IsDate()
+  readonly snoozedUntilDate?: Date | null;
```

Comment:
```suggestion
  readonly snoozedUntil?: Date | null;
```

Developers will know it's a date from its type.